tests/system/Commands/Utilities/NamespacesTest.php

```diff
@@ -37,21 +37,23 @@ protected function tearDown(): void
 
     protected function getBuffer()
     {
-        return $this->getStreamFilterBuffer();
+        return str_replace(PHP_EOL, "\n", $this->getStreamFilterBuffer());
     }
 
     public function testNamespacesCommandCodeIgniterOnly()
     {
         command('namespaces -c');
 
-        $expected = <<<'EOL'
+        $ds = DIRECTORY_SEPARATOR;
+
+        $expected = <<<EOL
             +---------------+-------------------------+--------+
             | Namespace     | Path                    | Found? |
             +---------------+-------------------------+--------+
-            | CodeIgniter   | ROOTPATH/system         | Yes    |
-            | App           | ROOTPATH/app            | Yes    |
-            | Config        | APPPATH/Config          | Yes    |
-            | Tests\Support | ROOTPATH/tests/_support | Yes    |
+            | CodeIgniter   | ROOTPATH{$ds}system         | Yes    |
+            | App           | ROOTPATH{$ds}app            | Yes    |
+            | Config        | APPPATH{$ds}Config          | Yes    |
+            | Tests\\Support | ROOTPATH{$ds}tests{$ds}_support | Yes    |
```

Comment:
I don't want to write test code like these.
Can't we write `ROOTPATH/tests/_support` as before?

Reply:
We could use a slash in the paths instead of DIRECTORY_SEPARATOR.
For Windows it won't matter (slash or backslash).